contents/teams/data-stack/objectives.mdx

```diff
@@ -1,21 +1,19 @@
 
-Objective: Data as a platform
+Objective: Managed data warehouse MVP
 
 * **Key Results:**
-    * Opinionated pattern for how data warehouse data is joined in other products
-    * Managed materialization and views
+    * Validate query engine alternatives to querying columnar data format
+    * Build and test an MVP of new format on internal team data
 
-Objective: Extensible sources
+Objective: Maturing the data pipeline
 
 * **Key Results:**
     * 10+ more sources shipped
     * Self-serve source creation
-    * Refactored to be backend config based
+    * Update billing to be user friendly (grace period on initial sync)
+    * Increased reliability and stability
 
-Objective: UX across data warehouse
+Objective: Warehouse integration across PostHog
 
 * **Key Results:**
-    * Data warehouse scene to manage all DW related resources
-    * Extended person properties UX
-    * Max + editor UX 
-    * AI autocomplete
```

Comment:
I have some ideas for this I want to work on as a side project. We don't have to make it a goal. Prob not the highest priority. Just commenting for visibility.